KubeArmor/go.mod

```diff
@@ -20,82 +20,81 @@ replace (
 )
 
 require (
-	github.com/cilium/ebpf v0.9.0
-	github.com/containerd/containerd v1.5.13
+	github.com/cilium/ebpf v0.9.1
+	github.com/containerd/containerd v1.6.8
```

Comment:
Just a doubt, folks (me included) are facing some issue in exec-ing into containers in containerd version > v1.6.6 (https://askubuntu.com/questions/1424317/docker-20-10-ubuntu-22-04-oci-runtime-exec-failed), will we face the same issue while ci-testing? Do our test cased exec commands inside the running containers?

Reply:
Yup, It has been fixed in upstream https://github.com/containerd/containerd/pull/7315 but ig they are yet to make a release. I am not sure if this would impact anything here. We are not execing into containers, just watching into events.

Reply:
@Ankurk99 Barun is right. We haven't updated the the test suite yet so it shouldn't affect the exec based test cases in karts. This updates the Containerd client we use in KubeArmor's container monitor.